RELEASING.md

```diff
@@ -2,30 +2,50 @@
 
 ## Preparing for the Release
```

Comment:
setting it up for the first time. Since this is already set up ...we should really refer to the releasing section.